srcpkgs/rtags/template

```diff
@@ -1,10 +1,12 @@
 # Template file for 'rtags'
 pkgname=rtags
-version=2.16
-revision=3
+version=2.18
```

Comment:
why don't you use the [tarball](https://github.com/Andersbakken/rtags/releases/tag/v2.18), instead of `git clone` ?

Reply:
Github's autogenerated tarballs don't include submodules IIRC

Reply:
yes, we know... but it is better to do that like in:
https://github.com/void-linux/void-packages/blob/master/srcpkgs/prometheus-cpp/template
https://github.com/void-linux/void-packages/blob/master/srcpkgs/albert/template
for example.

Reply:
Done, thanks for the info :)